wpilibc/Athena/src/DriverStation.cpp

```diff
@@ -291,8 +319,11 @@ int DriverStation::GetStickPOV(uint32_t stick, uint32_t pov) {
     wpi_setWPIError(BadJoystickIndex);
     return -1;
   }
-
+  std::unique_lock<priority_mutex> lock(m_joystickDataMutex);
   if (pov >= m_joystickPOVs[stick].count) {
+    // Unlock early so error printing isn't locked.
+    m_joystickDataMutex.unlock();
```

Comment:
It should be sufficient to simply call lock.unlock() here: http://en.cppreference.com/w/cpp/thread/unique_lock/unlock